openedx.yaml

```diff
@@ -0,0 +1,7 @@
+# This file describes this Open edX repo, as described in OEP-2:
+# http://open-edx-proposals.readthedocs.io/en/latest/oeps/oep-0002.html#specification
+
+nick: acct
+oeps: {}
+owner: edx/arch-team
+track-pulls: true
```

Comment:
Please delete this line.  This key is obsolete.

Reply:
Ah!  Thanks Ned.  Will apply back to the frontend-app-profile as well.